config/routes.d/staging.horse.json

```diff
@@ -6,7 +6,6 @@
       "^/docs/api-doc-template/": "developer.rackspace.com/user-guide.html",
       "^/docs/billing-api/": "developer.rackspace.com/user-guide.html",
       "^/docs/rkaas/": "developer.rackspace.com/user-guide.html",
```

Comment:
since you deleted the line below it, this will cause an extra comma on the `rkaas` line. Please remove the comma on the final line.

Reply:
Oh good catch :) Thank you